helm/charts/hydra-maester/values.yaml

```diff
@@ -26,7 +26,9 @@ adminService:
   port:
 
 forwardedProto:
-    
+
+syncPeriod: 10h
```

Comment:
i'd prefer to move the value to either `deployment.args.syncPeriod` or to be more in line with our other charts `maester.config.syncPeriod`

Reply:
I've moved it to deployment.args.syncPeriod.